crates/bls-crypto/src/hash_to_curve/try_and_increment.rs

```diff
@@ -100,6 +98,22 @@ where
             // produce a hash with sufficient length
             let candidate_hash = self.hasher.hash(domain, msg, hash_bytes)?;
 
+            // handle the Celo deployed bit extraction logic
+            cfg_if::cfg_if! {
```

Comment:
I think we can avoid the `cfg_if` dependency by doing

```rust
#[cfg(feature = "compat)]
let candidate_hash = {
     ....
};
```

Reply:
Nice! Removed it.